functions/Remove-SqlOrphanUser.ps1

```diff
@@ -246,7 +352,7 @@ Will remove from all databases the user OrphanUser EVEN if exists their matching
             Write-Output "There are no databases to analyse."
```

Comment:
I would avoid using throw, and rather catch errors and define the output ourselves, like with Write-Error